src/ai-bolit.php

```diff
@@ -2190,6 +2190,13 @@ function escapedDec($escaped)
 
 function UnwrapObfu($par_Content) {
   $GLOBALS['g_EncObfu'] = 0;
+  
+  $search  = array( ' ;', ' =', ' ,', ' .', ' (', ' )', ' {', ' }', '; ', '= ', ', ', '. ', '( ', '( ', '{ ', '} ');
+  $replace = array(  ';',  '=',  ',',  '.',  '(',  ')',  '{',  '}', ';',  '=',  ',',  '.',  '(',  ')',  '{',  '}');
+  $par_Content = str_replace('@', '', $par_Content);

```

Comment:
If malware signature contains @ character it will fail because of this. Shall we also remove @ from every single signature on start?